src/models/transaction.ts

```diff
@@ -92,6 +93,32 @@ class Transaction {
     this._dataToSignCache = null;
   }
 
+  /**
+   * Get the signature from the dataToSignHash for a private key
+   *
+   * @return {Buffer}
+   * @memberof Transaction
+   * @static
+   *
+   */
+  static sign(dataToSignHash: Buffer, privateKey: PrivateKey): Buffer {
```

Comment:
Why do you have this method as well? You could use only the other one, the static method is the same as the utils method, right?

Reply:
Yes, it´s exactly the same method. Just kept it here for convenience, do you think we should remove to keep this cleaner?

Reply:
I feel we could remove it and keep only the other one.